iml-agent/src/action_plugins/lctl.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{
+    agent_error::ImlAgentError,
+    cmd::{self},
```

Comment:
```suggestion
    cmd,
```

Reply:
Don't I need cmd::{self} if I reference cmd?

Reply:
It should work without it.

Reply:
`self` is just a helper if you are importing other things under that namespace and still want to use `cmd` as well.